vsts/cleanup.ps1

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+Function IsWindows()
+{
+	return ([Environment]::OSVersion.Platform -eq [System.PlatformID]::Win32NT)
+}
+
+if (IsWindows)
+{
+	Write-Host Start ETL logging
+	logman create trace IotTrace -o iot.etl -pf tools/CaptureLogs/iot_providers.txt
+	logman start IotTrace
+}
+
+Write-Host List installed .NET SDK versions
+dotnet --list-sdks
+
+.\build.ps1 -runCleanupSamples
```

Comment:
This is probably the only line of code required. We might be able to put that in the yaml itself.